### PR TITLE
Issue #11163: Enforce file size for FinalLocalVariable

### PR DESCRIPTION
**Pull request by Rohanraj123**
Part of #11163

### CODE REVIEW

**Comment by Rohanraj123** [comment]
Can we please put HacktoberFest accepted label over this PR. @romani

**Review by romani — APPROVED** [review]
Ok to merge

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/finallocalvariable/InputFinalLocalVariableFive.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                }
+            }
+
+            abstract class Bar2 {
+                abstract void method(String param);
+            }
+        }
+
+        public void test5() {
+InputFinalLocalVariableFive table = new InputFinalLocalVariableFive(); // violation
```
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/finallocalvariable/InputFinalLocalVariableFive.java:81]: Same.

**Review by rnveach — CHANGES_REQUESTED** [review]
The commit of this PR is misleading.
> Enforce Java file for FinalLocalVariable

This sounds like the issues where we are converting TXT file in examples to Java files.

Please rename to `Issue #11163: Enforce file size for FinalLocalVariable`.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/finallocalvariable/InputFinalLocalVariableTwo.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            int k = 0; // violation
+            {
+                l++;
+            }
+        }
+
+        int weird = 0;
+        weird++;
+
+final InputFinalLocalVariableTwo.InnerClass ic = new InputFinalLocalVariableTwo.InnerClass();
```
[the remark this answers is not in the document]
Comment (Rohanraj123) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/finallocalvariable/InputFinalLocalVariableTwo.java]: Done with the changes @rnveach Can you please review it once .

**Review by rnveach — APPROVED** [review]

**Comment by MauryanKansara** [comment]
I'm working on NoEnumTrailingComma.

**Comment by romani** [comment]
@MauryanKansara, please post this in issue, not a Pull Request.

**Pull request merged**